client/src/commands/index.ts

```diff
@@ -2,60 +2,69 @@ import {
   sendEvent, 
 } from "../network"
 import {
-  CHAT_EVENT, 
-  INPUT_EVENT, 
-  LOG_EVENT,
+  CHAT_EVENT, INPUT_EVENT, LOG_EVENT, 
 } from "../../../events"
 import {
   commandEmitter, 
 } from "./emitter"
-import Roll from './roll'
+import Roll from "./roll"
 import {
   MakeRoom, 
 } from "./makeRoom"
 import {
-  Go,
+  Go, 
 } from "./go"
 import {
   MESSAGE_MAX_LENGTH, 
 } from "../../../constants"
 import {
-  LogItem,
+  LogItem, 
 } from "../components/Terminal"
 import {
   Help, 
 } from "./help"
+import Nick from "./username"
 
-export const commandModules = [
-  Roll,
-  MakeRoom,
-  Go,
-  Help,
-]
+export const commandModules = [Roll, MakeRoom, Go, Help, Nick]
```

Comment:
I think this should remain multiline as we'll eventually have loads of commands. This helps cut down on merge conflicts.